pages/common/ffmpeg.md

```diff
@@ -4,7 +4,7 @@
 
 - Extract the sound from a video and save it as MP3:
 
-`ffmpeg -i {{video.mp4}} {{sound}}.mp3`
+`ffmpeg -i {{video.mp4}} -vn -codec:audio {{sound}}.mp3`
```

Comment:
We always try to give the simplest possible example as the first one. Is this absolutely necessary ?

Reply:
I'd argue that the -vn is necessary as it's not uncommon to wind up with an MP3 file where ffmpeg has taken a guess and embedded something it can convert to an image out of the video stream into to tags.

Although you are right that specifying audio codec is probably superfluous.